### PR TITLE
Documentation - Internationalization

### PR DESCRIPTION
**Pull request by KasunApplications**
Add documentation for adding languages #203

### CODE REVIEW

**Comment by KasunApplications** [comment]
Pls review @sudheesh001

File: docs/Internationalization.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+### Step 1
+
+Add prefered language to application as follows
+
+![screen shot 2017-08-17 at 9 30 54 pm](https://user-images.githubusercontent.com/16130948/29444416-0b8868b2-83fd-11e7-91f0-330c11dced40.png)
+ 
+ 
+ ### Step 2
+ 
+ Add tranlation of language to the followng file
```
Comment (sudheesh001) [inline docs/Internationalization.md]: typo. `Translation`

File: docs/Internationalization.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+## Internationalization - Adding Languages
+
+For Internationalization with additional languages we have to follow two steps
+
+### Step 1
+
+Add prefered language to application as follows
```
Comment (sudheesh001) [inline docs/Internationalization.md]: typo: `preferred`

File: docs/Internationalization.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+## Internationalization - Adding Languages
+
+For Internationalization with additional languages we have to follow two steps
```
Comment (sudheesh001) [inline docs/Internationalization.md]: correct it to `follow the steps given below`

File: docs/Internationalization.md (hunk 1 above)
Comment (sudheesh001) [inline docs/Internationalization.md]: typo. `following`

**Comment by sudheesh001** [comment]
Squash commits into one please

File: docs/Internationalization.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
+## Internationalization - Adding Languages
+
+Follow the steps given below
+
+### Step 1
+
+Add preferred language to application as follows
+
+![screen shot 2017-08-17 at 9 30 54 pm](https://user-images.githubusercontent.com/16130948/29444416-0b8868b2-83fd-11e7-91f0-330c11dced40.png)
```
Comment (isuruAb) [inline docs/Internationalization.md]: If you can mention how and where we can get this window  It would be really nice for other users.

**Pull request merged**